README.md

```diff
@@ -38,7 +38,24 @@ or downloading a file if it already exists with the same content locally or in t
 ## Installation
 
 An app bundle is provided for macOS High Sierra and higher and can be downloaded from the
-Releases tab. On other platforms, please download and install the Python package from PyPI:
+Releases tab. This app Bundle is also package as a Homebrew cask.
```

Comment:
```suggestion
An app bundle is provided for macOS High Sierra and higher. It can be downloaded from the
[Releases](https://github.com/SamSchott/maestral/releases) tab and is also packaged as a Homebrew cask.
```

I would restructure those sentences to have the download information bundled in one sentence and the packaging format in another.